xbmc/guilib/GUIControlGroupList.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "GUIControlGroupList.h"
 #include "input/Key.h"
-#include "GUIInfoManager.h"
+#include "guiinfo/GUIInfoLabels.h"
```

Comment:
What @da-anda said and it's not really part of guilib even if it's heavily used there